src/components/Common/FilePreviewDialog.tsx

```diff
@@ -135,6 +190,14 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
             </div>
           </div>
           <div className="flex flex-1 items-center justify-center">
+            {uploadedFiles && uploadedFiles.length > 1 && index > 0 && (
```

Comment:
Let's remove the last condition; instead disable the button if index is 0, rather than hiding it.

Vice versa for the other button.

Reply:
done